web/client/components/catalog/SharingLink.jsx

```diff
@@ -32,14 +33,16 @@ const SharingLink = React.createClass({
         if (!this.props.url) {
             return null;
         }
+        // add authentication to the url if possible
+        const url = SecurityUtils.addAuthenticationToUrl(this.props.url);
```

Comment:
Method SecurityUtils.addAuthenticationToUrl will check if authentication is activated before adding anything.